src/metrics/prometheus.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+// Copyright (c) 2018 Jaeger Author.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+import { Counter as PromCounter, Gauge as PromGauge } from 'prom-client';
+
+let CounterPromWrapper = class {
+  _counter: PromCounter;
+  _labels: any;
+
+  constructor(counter: PromCounter, labels: any) {
```

Comment:
can we minimize the use of `any`? it's avoiding Flow type checks.

Reply:
I replaced `any` by `{}` which is for arbitrary object type. More type checking can be done if flow version is upgraded such as [Objects as maps](https://flow.org/en/docs/types/objects/#toc-objects-as-maps) for `_cache: any = {};`